.github/workflows/evolve.yml

```diff
@@ -9,7 +9,6 @@ jobs:
   # TODO: add unit tests using bused
   evolve:
```

Comment:
just but this in the build/test evolve and add an if check for the branch. similar to https://github.com/ar-io/ar-io-network-process/blob/develop/.github/workflows/build.yaml#L59-L74